app/models/node.rb

```diff
@@ -383,7 +383,11 @@ def parent_table_node
   end
 
   def preload_tree(children)
-    ActiveRecord::Associations::Preloader.new.preload(children, :child_nodes)
+    if Rails.version =~ /\A6\.1/
```

Comment:
just FYI. This deprecation came up when I ported to standalone Rails 7 app for schema development. I'm pretty confident this code isn't used by the DMS webapp.